pycbc/filter/qtransform.py

```diff
@@ -0,0 +1,254 @@
+# Copyright (C) 2017  Hunter A. Gabbard
+# Most of this is a port of Duncan Macleod's GWPY qtransform.py script
```

Comment:
@duncan-brown No, not really. We'd like to integrate this so we can use it with the strain module directly and have it as part of the standard pycbc_inspiral type applications to generate information to use within the ranking statistic. GWPy pulls in a huge number of dependencies and has a lot of code duplication at this point. One solution I floated was to spin off the qtransform, but nobody seemed keen on that idea.